nautobot/extras/jobs.py

```diff
@@ -919,11 +1040,8 @@ def is_job(obj):
     """
     Returns True if the given object is a Job subclass.
```

Comment:
Would be nice for this to be replaced by a `obj.Meta.abstract` check, per my earlier comment.